wizard/src/utils/detect_place.ts

```diff
@@ -213,13 +216,19 @@ export class PlaceDetector {
     this.stateNames = new Set<string>();
```

Comment:
nit: is this function more general than preProcessUSStates?  Given isoCodes exist everywhere?